packages/ui/src/lib/table/Table.svelte

```diff
@@ -25,17 +25,27 @@ export let data: { selected?: boolean; name?: string; id?: any }[];
 export let defaultSortColumn: string | undefined = undefined;
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let expanded: any[] = [];
+export let collapsed: any[] = [];
```

Comment:
Do we really need the any here ?

Reply:
I assume you mean 'does it really need to be 'any' or should it be a string'?

It was 'any' because originally I was storing the row object itself, which turned out to be a bad idea b/c the UI objects are ephemeral, then I switched it to requiring that any object you want to collapse must have an id. I think it's reasonable enough and fits our cases that the id is a string, so I've switched it in another commit.

Reply:
Okey good!